wild-swimming-app/src/feature/map/utils/getUserLocation.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+
+const useGeoLocation = () => {
+  const [location, setLocation] = useState({
+    loaded: false,
+    coordinates: { lat: 0, lng: 0 },
+  });
+
+  const onSuccess = (location) => {
+    setLocation({
+      loaded: true,
+      coordinates: {
+        lat: location.coords.latitude,
+        lng: location.coords.longitude,
+      },
+    });
+  };
+
+  //   const onError = (error) => {
+  //     setLocation({
```

Comment:
and commented out code - is this being kept for just incase when we bring in other components and functionality?